src/app/feed/feed-card/feed-card.component.scss

```diff
@@ -108,8 +108,9 @@ a.post-link {
 	background-color: rgba(0, 0, 0, 0.5);
 
 	.overlay-content{
+		max-width: 90%;
+		height: 340px !important;
```

Comment:
Why fixing the height to 340px ?? This is not responsive. The content will overflow.